.changeset/brown-trainers-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": minor
+---
+
+Refactor styling and text of `DeleteDialog` in `CrudContextMenu`
```

Comment:
"Refactor" technically isn't correct, as this changes the result 🙂

Reply:
```suggestion
Update styling and text of `DeleteDialog` in `CrudContextMenu`
```

Reply:
```suggestion
Adapt the `DeleteDialog` in `CrudContextMenu` to match the updated Comet CI
```